### PR TITLE
[DEVOPS-1207] Project configuration list moved to angular / Toggle an…

### PR DESCRIPTION
**Pull request by jhonypg**
…d Legend Component created

### CODE REVIEW

File: core/src/main/java/objective/taskboard/project/config/ProjectConfigurationController.java [hunk 1]
```diff
@@ -99,11 +99,13 @@ public ProjectConfigurationController(
 
     public static class ProjectListItemDto {
         public String projectKey;
-        public Boolean isArchived;
+        public String projectName;
```
Comment (objective-solutions-user) [inline core/src/main/java/objective/taskboard/project/config/ProjectConfigurationController.java]: MINOR Make projectName a static final constant or non-public and provide accessors if needed. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)

File: core/src/main/java/objective/taskboard/project/config/ProjectConfigurationController.java [hunk 2]
```diff
@@ -99,11 +99,13 @@ public ProjectConfigurationController(
...
+        public boolean isArchived;
```
Comment (objective-solutions-user) [inline core/src/main/java/objective/taskboard/project/config/ProjectConfigurationController.java]: MINOR Make isArchived a static final constant or non-public and provide accessors if needed. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: application/src/test/java/objective/taskboard/it/config/project/ProjectList.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    private WebElement findProjectKeyEl(ProjectInfo projectInfo) {
+        return getProjectList().stream()
+                .map(pEl -> pEl.findElement(cssSelector(PROJECT_KEY_SELECTOR)))
+                .filter(pKeyEl -> projectInfo.key.equals(pKeyEl.getText()))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException("Element for project key " + projectInfo.key + " filter not found"));
+    }
+
+    private ProjectList assertProjectList(String... expectedList) {
+        waitAssertEquals(join(expectedList, "\n"), () -> {
```
Comment (mauricio-frontend) [inline application/src/test/java/objective/taskboard/it/config/project/ProjectList.java]: My doubt hear if is not to be good to use a array asserter hear?

File: core/web-app/src/tslint.json [hunk 4]
```diff
@@ -14,6 +14,7 @@
             "kebab-case"
         ],
         "use-host-property-decorator": false,
-        "no-input-rename": false
+        "no-input-rename": false,
+        "member-ordering": false
```
Comment (mauricio-frontend) [inline core/web-app/src/tslint.json]: What means this property, just to understand?

**Pull request merged**